vignettes/report-writing-vignette.Rmd

```diff
@@ -0,0 +1,380 @@
+---
+title: "Report Writing"
+output: 
```

Comment:
I think this is going to cause errors. I think it should be

```yaml
output: rmarkdown::html_vignette
```

Instead of 

```yaml
output:
  rmarkdown::html_vignette:
```

You shouldn't have to mess with the YAML front matter for vignettes, other than the vignette title. See [here](http://r-pkgs.had.co.nz/vignettes.html) for more vignette info.